core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksActivity.java

```diff
@@ -206,7 +207,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
         allBookmarks.clear();
         bookmarksList.clear();
         bookmarksAdapter.notifyDataSetChanged();
-        Toast.makeText(this, R.string.all_bookmarks_cleared_toast, Toast.LENGTH_SHORT).show();
+        Snackbar.make(getWindow().getDecorView().getRootView(), R.string.all_bookmarks_cleared, Snackbar.LENGTH_SHORT).show();
```

Comment:
you couldn't use `getView()`?

Reply:
I tried that. It didn't work, so I had to use this.

Reply:
Use any of the views provided by butterknife, I had a brainfart, Activities do not have getView.

The way snackbar works is it traverses up the view heirarchy for the nearest CoordinatorLayout and then plops the snackbar into it, hopefully moving views with the right behaviours, ie FABs.